lua/fishing_mod/catch/stove.lua

```diff
@@ -274,7 +289,7 @@ else
 	
 		if (self.next_search or 0) < CurTime() then
 		
-			for key, entity in pairs( ents.FindInBox( self:GetPos() + self:OBBMins(), self:GetPos() + self:OBBMaxs() ) ) do
+			for key, entity in pairs( ents.FindInBox( self:GetPos() + self:OBBMins(), self:GetPos() + Vector(self:OBBMaxs().x, self:OBBMaxs().y, self:OBBMaxs().z * 1.2 ) ) ) do

```

Comment:
multiply the vector itself the operator is implemented for scalars and you're creating a whole new vector for nothing